kite-morphlines/kite-morphlines-hadoop-rcfile/src/test/java/org/kitesdk/morphline/hadoop/rcfile/ReadRCFileTest.java

```diff
@@ -122,8 +123,25 @@ public void testRCFileColumnWise() throws Exception {
         false));
   }
 
+  @Test
+  public void testRCFileWithNull() throws Exception {
```

Comment:
This differs from `testRCFileRowWise` by only one line. Can we pull the common parts into a helper method?